kornia-viz/src/bin/app.rs

```diff
@@ -27,7 +27,8 @@ struct KorniaApp {
 
 impl Default for KorniaApp {
     fn default() -> Self {
-        let capture = kornia::io::stream::V4L2CameraConfig::new()
+        // Use NVCameraConfig instead of V4L2CameraConfig for GPU acceleration
+        let capture = kornia::io::stream::NVCameraConfig::new()
```

Comment:
I would make a new example instead using rerun for now. The kornia-viz was a dead end experiment which probably I'll remove soon as it need to much maintainership